src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java

```diff
@@ -19,6 +19,7 @@ public class Device extends AbstractModelObject {
   private final String name;
   private final String type;
   private final Integer volume_percent;
+  private final Boolean supports_volume;
```

Comment:
Would you mind keeping the alphabetical sorting of properties here and in other places?

Reply:
No, I can do :).

But what be "other places", just `Device.java`?

Reply:
Yes

Reply:
Done